platform-ajax/src/main/java/com/softicar/platform/ajax/customization/IAjaxStrategy.java

```diff
@@ -76,17 +76,17 @@ public interface IAjaxStrategy {
 	// -------------------- sessions -------------------- //
 
 	/**
-	 * Tests whether the given {@link HttpSession} is a super-user session.
+	 * Tests whether the given {@link HttpSession} is a administrator session.
 	 * <p>
-	 * Some features like debug mode, etc. are only available to super-user
+	 * Some features like debug mode, etc. are only available to administrator
 	 * sessions.
 	 *
 	 * @param session
 	 *            the {@link HttpSession} to test (never <i>null</i>)
-	 * @return <i>true</i> if the given {@link HttpSession} is a super-user
+	 * @return <i>true</i> if the given {@link HttpSession} is a administrator
 	 *         session; <i>false</i> otherwise
 	 */
-	boolean isSuperUser(HttpSession session);
+	boolean isAdministrator(HttpSession session);
```

Comment:
In order to not wrongfully hint to a (nonexistent) "administrator" role, we could rename this method to `isAdministrative`. So if `true`, we got an "administrative session". Seems kind of natural.